src/app/payment/payment-content.tsx

```
@@ -17,61 +32,74 @@ import { maskEmail, maskPhoneNumber } from "@/lib/utils/mask-data";
-        }`}
+    <motion.div
+      whileHover={{ scale: 1.02 }}
+      whileTap={{ scale: 0.98 }}
+      className="w-full"
+    >
+      <div
+        className={`p-4 cursor-pointer transition-all duration-300 bg-white rounded-xl border ${
+          selected ? "border-green-500 bg-green-50/50" : "border-gray-100"
+        } hover:shadow-md`}
         onClick={onClick}
       >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)